internal/controllers/clusterextension_admission_test.go

```diff
@@ -222,11 +226,13 @@ func TestClusterExtensionAdmissionInstallNamespace(t *testing.T) {
 			t.Parallel()
 			cl := newClient(t)
 			err := cl.Create(context.Background(), buildClusterExtension(ocv1alpha1.ClusterExtensionSpec{
-				PackageName:      "package",
-				InstallNamespace: tc.installNamespace,
+				PackageName:        "package",
+				InstallNamespace:   tc.installNamespace,
+				ServiceAccountName: tc.serviceAccountName,
```

Comment:
This test is just for the `spec.installNamespace` validation. Should we hardcode `ServiceAccountName` to something we know will be valid?

That would avoid maintainer/contributor confusion. Folks might see that the service account name is varied as part of this test and conclude that the service account name somehow plays into the installNamespace validatation.

We also want to test side-effects from varying service account names.

Reply:
agreed